tests/unit/channel_tests.py

```diff
@@ -549,7 +559,7 @@ def test_close_in_opening_state(self, _unused):
             spec.Channel.Close(200, 'Got to go', 0, 0), self.obj._on_closeok,
             [spec.Channel.CloseOk])
 
-        self.assertEqual(self.obj._closing_code_and_text, (0, ''))
+        self.assertEqual(self.obj._closing_code_and_text, (200, 'Got to go'))
```

Comment:
`Channel.close()` member function now saves the passed-in values in `_closing_code_and_text`, so comparing against the bogus `(0, '')` no longer works.